Makefile

```diff
@@ -59,8 +62,6 @@ PROXY_ARGS := $(shell if [ "$$http_proxy" != "" ]; then echo "-e http_proxy=$$ht
 PROXY_ARGS += $(shell if [ "$$https_proxy" != "" ]; then echo " -e https_proxy=$$https_proxy"; fi)
 PROXY_ARGS += $(shell if [ "$$no_proxy" != "" ]; then echo " -e no_proxy=$$no_proxy"; fi)
 
-INSTALL_DEPS = 1
```

Comment:
Did you mean to remove `INSTALL_DEPS`?  I believe this is here so we can get around Jenkins not building golint but still allow users to call it out.

Reply:
oops, good catch James! Didn't mean to drop it, it's just miss in carrying the file from the internal repo. Putting it back now

Reply:
ok, INSTALL deps is back